packages/v4/src/content/design-guidelines/usage-and-behavior/alerts-and-notifications/alerts-and-notifications.md

```diff
@@ -9,119 +9,124 @@ Alerts are used when the system wants to notify the user about some status or ot
-
 ### Stacking multiple alerts
 Multiple alerts will stack. The most recent alert should appear at the top and others will push down. When alerts are deleted, any space above will be filled by pushing remaining alerts up in the stack order.
 
-An Alert Group component allows a set of alerts to be properly positioned and stacked as shown below.
-
-![alert group](./img/alert-group.png)
-
-### Modal alerts
-When a condition occurs that requires immediate action, a modal message dialog is recommended.
+An [Alert Group](documentation/react/components/alertgroup) component allows a set of alerts to be properly positioned and stacked as shown below.
 
-![modal notification](./img/modal-notification.png)
-
-For more information about designing modal messages see the [message dialog design guidelines](/design-guidelines/usage-and-behavior/modal).
+![alert group](./img/toast-notification-multiple.png)
 
 ## Content guidelines
 Message text should be succinct and clearly state what the issue or problem is. Embedded links should navigate the user to the location where they can take action.
 
-Alert notifications present special challenges for accessibility. Screen readers will announce the arrival and message content of an incoming toast alert, but user focus will not be placed there. This will make it difficult for these users to directly act on the alerts itself. Therefore we recommend that any message containing actions be descriptive of how the action can be performed in the application. Messages that contain links should specify the location in text as well as including a simple text link. For example, a simple notification message might take the form:
+Toast alerts present special challenges for accessibility. Screen readers will announce the arrival and message content of an incoming toast alert, but user focus will not be placed there. This will make it difficult for these users to directly act on the alerts itself. Therefore, we recommend that any message containing actions be descriptive of how the can navigate to perform the action in the application. Messages that contain links should specify the location in text as well as including a simple text link. For example, a simple notification message might take the form:
 
-![non-accessible toast notification](./img/notification-not-accessible.png)
+<img src="./img/Notification-content-not-accessible.png" alt="Non-accessible alert example" width="615"/>
 
 A more accessible version of this message would look like this:
 
-![non-accessible toast notification](./img/notification-accessible.png)
+<img src="./img/Notification-content-accessible.png" alt="Non-accessible alert example" width="615"/>
 
 While this is more verbose, it has the advantage of being usable for both sighted and non-sighted users by providing instructions to screen reader users about where they can go to complete the action.
 
+## Examples
```

Comment:
Instead of having this as a separate section, I wonder if it would be more useful/visible to add a "Placement" section under Inline Alerts and Toast alerts where you would add in these images and descriptions. Just because this could be easily missed!

Reply:
I moved the examples to be within the usage section but kept it labeled "Examples."

Reply:
I meant that I would split this example section into two, and add each part to the respective alert types' section, ie: have an "example" section under Inline alerts, and an "example" section under toast alerts so that the examples are closer to the respective alert details! But if you think this makes more sense that's fine too!